pkgs/tools/virtualization/cloudmonkey/default.nix

```diff
@@ -0,0 +1,25 @@
+{ python2Packages, lib }:
```

Comment:
I'm mostly a python user rather than a developer and couldn't find where to enable the tests for cloudmonkey (nothing in the repo or its [setup.py](https://github.com/apache/cloudstack-cloudmonkey/blob/master/setup.py)). Any pointers?